troubleshooting.md

```diff
@@ -0,0 +1,6 @@
+# TROUBLESHOOTING
```

Comment:
Hopefully this file won't be necessary, once the C API matters are sorted out. We will just need to specify the specific version of Faiss required in the readme.

Reply:
Can we check version faiss in code?

Reply:
I'm afraid not. The C API does not re-export the Faiss version declarations, although maybe it should.